### PR TITLE
exam

### PR DESCRIPTION
**Pull request by MuhammadTamzid**

### CODE REVIEW

**Comment by MuhammadTamzid** [comment]
Dear sir, it took times for this reason send Cobain two task in one pull request.
Thank you (bow)

File: app/controllers/lessons_controller.rb [hunk 1]
```diff
@@ -12,14 +12,32 @@ def create
   end
 
   def edit
+    @words = @lesson.category.words.sample(10)
```
Comment (MuhammadTamzid) [inline app/controllers/lessons_controller.rb]: will remove ()

File: app/controllers/lessons_controller.rb [hunk 2]
```diff
@@ -12,14 +12,32 @@ def create
...
-    params.require(:lesson).permit :category_id, :user_id
+    params.require(:lesson).permit :category_id, :user_id,
+      lesson_words_attributes: [:id, :word_id, :word_answer_id]
   end
 
   def load_lesson
     @lesson = Lesson.find params[:id]
   end
+
+  def init_lesson_words words
```
Comment (dieunb) [inline app/controllers/lessons_controller.rb]: You should add words to lesson on callback

**Comment by sujoydatta** [comment]
ok

**Pull request merged**